x-pack/plugins/remote_clusters/index.js

```diff
@@ -44,7 +44,13 @@ export function remoteClusters(kibana) {
         enabled: Joi.boolean().default(true),
       }).default();
     },
-
+    isEnabled(config) {
+      return (
+        config.get('xpack.remote_clusters.enabled') &&
+        config.has('xpack.index_management.enabled') &&
+        config.get('xpack.index_management.enabled')
```

Comment:
Can we remove line 51 since it's a dupe of line 50?

Reply:
it's not AFAIK

Reply:
OK, turns out the "has" calls are not necessary.  Saw this in someone else's code and assumed they were, but this doesn't crash Kibana: `config.get('xpack.doesntexist.enabled')`

Reply:
I actually had just scanned the code too quickly and didn't notice one was `has` and one was `get`. Looks like we both learned something new 😄